pyproject.toml

```diff
@@ -150,7 +150,24 @@ ignore = [
     "PIE790",
     "PLR2004",
     "PTH207",
-    "RUF012"
+    "RUF012",
+    "UP027",  # deprecated
+    "UP038",  # https://github.com/astral-sh/ruff/issues/7871
+    # https://docs.astral.sh/ruff/formatter/#conflicting-lint-rules
+    "W191",
+    "E111",
+    "E114",
+    "E117",
+    "D206",
+    "D300",
+    "Q000",
+    "Q001",
+    "Q002",
+    "Q003",
+    "COM812",
+    "COM819",
+    "ISC001",
+    "ISC002"
```

Comment:
OK with me but can I ask you to sort those alpha-numerically for the sake of our common long term mental heatlh. 🙏🏽

Reply:
I kept the order of the documentation. That's something I can change of course. Is it OK to keep the conflicting rules as a separate section?

```
    .
    .
    .
    "UP027",  # deprecated
    "UP038",  # https://github.com/astral-sh/ruff/issues/7871
    # https://docs.astral.sh/ruff/formatter/#conflicting-lint-rules
    "COM812",
    "COM819",
    .
    .
    .
    "W191"
```

Reply:
YES I was thinking to actually keep that comment as a separator so we know which ones come from us or not: don't hesitate to make that comment even more clear about this !!!

Reply:
Done.